src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserSignedIn?: boolean
```

Comment:
I have another better name. How about `isLoggedIn`?

Reply:
I personally name variables specifically since there will be no room for confusions. Can we go with something more specific like `isUserLoggedIn`?

PS: I only use `isLoggedIn` only when it is followed by a subject like `User.isLoggedIn`

Reply:
Or else I can make an interface name User and use your suggestion (`isLoggedIn`)
```
interface User {
  isLoggedIn: boolean
} 
```

Reply:
In my opinion, `LoggedIn` relates to `Log in` button on our design. Anyway, either `isLoggedIn` or `isUserLoggedIn` looks fine to me.

Reply:
Lets keep it like this first by just renaming 👍